[Email Routing] New UI (#4243)
cloudflare/cloudflare-docs: content/email-routing/get-started/_index.md

---
title: Get started
pcx-content-type: get-started
weight: 2
---

# Get started

> To enable Email Routing, start by creating a custom email address linked to a destination address. This forms a **rule**. You can toggle rules on or off from the Clouflare dashboard. Refer to [Enable Email Routing](/email-routing/get-started/enable-email-routing) for more details.

Reviewer: ```suggestion
To enable Email Routing, start by creating a custom email address linked to a destination address. This forms a **rule**. You can enable or disable rules from the Cloudflare dashboard. Refer to [Enable Email Routing](/email-routing/get-started/enable-email-routing) for more details.
```
Probably just a matter of personal preference... feel free to disregard the first part. However, fixing the typo is important. :)

The first time you access Email Routing, you will see a wizard guiding you through the process of creating rules. You can skip the wizard and add rules manually.

> If you need to pause Email Routing or offboard to another service, refer to the [Settings page](/email-routing/get-started/settings) where you can find the options to do so.

Reviewer: ```suggestion
If you need to pause Email Routing or offboard to another service, use the available options in the [Settings page](/email-routing/get-started/settings/).
```
